apps/dashboard/src/app/team/[team_slug]/[project_slug]/insight/components/BlueprintsPage.tsx

```diff
@@ -0,0 +1,62 @@
+import { Spinner } from "@/components/ui/Spinner/Spinner";
+import { ClientOnly } from "components/ClientOnly/ClientOnly";
+import { BillingAlerts } from "components/settings/Account/Billing/alerts/Alert";
+import { Suspense } from "react";
+import { type Blueprint, BlueprintsExplorer } from "./BlueprintsExplorer";
+import { BlueprintsPageHeader } from "./BlueprintsPageHeader";
+
+export function BlueprintsPage(props: {
+  className?: string;
+}) {
+  return (
+    <div className={props.className}>
+      <BillingAlerts className="mb-10" />
+      <BlueprintsPageHeader />
+      <div className="h-6" />
+      <Suspense fallback={<Loading />}>
+        <BlueprintsPageAsync />
+      </Suspense>
+    </div>
+  );
+}
+
+function getProjectBlueprints() {
+  return [
+    {
+      id: "1",
+      name: "Transactions",
+      slug: "transactions-blueprint",
+      description: "Query transaction data",
+    },
+    {
+      id: "2",
+      name: "Events",
+      slug: "events-blueprint",
+      description: "Query event data",
+    },
+    {
+      id: "3",
+      name: "Tokens",
+      slug: "tokens-blueprint",
+      description: "Query ERC-20, ERC-721, and ERC-1155 tokens",
+    },
+  ] as Blueprint[];
+}
+
+async function BlueprintsPageAsync() {
```

Comment:
This component does not need to be async

Reply:
removed async